README.md

```diff
@@ -150,18 +110,59 @@ Create a `Scrabble::TileBag` class with __full unit testing/specs__. You should
 | L : 4  | Y : 2 |
 | M : 2  | Z : 1 |
 
-#### Modifications to `Player`
-Create specs for and add to the `Player` class the following instance methods:
+There is no requirement for the program to change this letter pool over time. Imagine that every time this method is called, it is a new letter pool every time.
+
+### Wave 2
+
+Implement the following method in the file `adagrams.rb`:
+
+- `uses_available_letters? input, letter_bank`
+  - This method should check if the input word only uses characters that are contained within the passed in letter bank. Essentially, this method checks if the word is, indeed, an anagram of some or all of the given letters
+  - Has two parameters:
+     - `input`, the first parameter, describes some input word, and is a string
+     - `letter_bank`, the second parameter, describes an array of drawn letters. You can expect this to be an array of ten strings, with each string representing a letter
+  - Returns either `true` or `false`
+  - Returns `true` if every letter in the `input` word is available (in the right quantities) in the `letter_bank`
+  - Returns `false` if not; if there is a letter in `input` that is not present in the `letter_bank` or has too much of compared to the `letter_bank`
+
+### Wave 3
+
+Implement the following method:
+
+- `score_word word`
+  - This method returns the score of a given word as defined by the Adagrams game
+  - Has one parameter: `word`, which is a string of characters
+  - Returns an integer representing the number of points
```

Comment:
Could we change the name of this method to just `score`? Someone once told me it's bad practice to have the last word of your method name be the same as the first parameter name.